packages/myst-to-jats/src/index.ts

```diff
@@ -848,9 +848,14 @@ export function writeJats(file: VFile, content: ArticleContent, opts?: DocumentO
     : doc.body();
   const jats = js2xml(element, {
     compact: false,
-    spaces: opts?.spaces,
+    spaces: opts?.spaces === 'flat' ? 0 : opts?.spaces || 1,
```

Comment:
Ah, I see - it looks like there's not actually a way to write xml with new lines but no indentation with `js2xml`. If you use `0` or `''`, you get a single line.

So we are deviating from the `js2xml` api by making `flat` a single line, and `0` multiple lines with no indentation. 👍

Reply:
Yeah, code could have been clearer. I will add a comment!

Reply:
```suggestion
    //  No way to write XML with new lines, but no indentation with js2xml.
    // If you use 0 or '', you get a single line.
    spaces: opts?.spaces === 'flat' ? 0 : opts?.spaces || 1,
```